### DISCUSSION TITLE
Is there any simple way to edit the main index.html...?

### DISCUSSION THREAD

**Discussion by taneemishere** [ppoffice/hexo-theme-icarus#1033] [Q&A] [answered]
Is there any simple way to edit and changes tags and their properties in the main index.html file as because every time I do generate the files and put it on server via the commands `hexo generate` and `hexo server` the changes disappear.

**Comment by ppoffice** [comment]
You are not supposed to change HTML directly. Change the layout files instead.

**Reply by taneemishere** [reply]
Then can you please guide me that how can I change the things like; `author_title` to multiple line and also how can I change fonts style to some others rather than the default one.

**Comment by ppoffice** [comment, marked as answer]
1. Edit `profix.jsx` https://github.com/ppoffice/hexo-theme-icarus/blob/87fb6f2c991d0eb079c86fdafd384c9c13b9a88f/layout/widget/profile.jsx#L41
  
    ```diff
    - {authorTitle ? <p class="is-size-6 is-block">{authorTitle}</p> : null}
    + {authorTitle ? <p style="white-space: pre-wrap" class="is-size-6 is-block">{authorTitle}</p> : null}
    ```

2. Edit `_config.yml` of your theme (or _config.icarus.yml):

    ```yml
    author_title: |
        line 1
        line 2
    ```

**Comment by taneemishere** [comment]
@ppoffice hey thanks for the upper response as it helped me to add multiple author titles to my bio. **Now one thing I'm experiencing an issue is that I can't find any way to add some other fonts to my site, or replace the default with my fonts**, I followed a discussion here in this [link](https://github.com/ppoffice/hexo-theme-icarus/issues/307) but seems it has been changes since then, as it is from 2018. So would be great if guide me. Thanks in advance.
